extern/cfe/src/bmi_cfe.c

```diff
@@ -956,26 +926,42 @@ static int Get_var_itemsize (Bmi *self, const char *name, int * size)
 }
 
 
-// TODO: complete implementation
-static int Get_var_location(Bmi *self, const char *name, char *location)
+static int Get_var_location (Bmi *self, const char *name, char * location)
 {
+    // Check to see if in output array first
+    for (int i = 0; i < OUTPUT_VAR_NAME_COUNT; i++) {
+        if (strcmp(name, output_var_names[i]) == 0) {
+            strncpy(location, output_var_locations[i], BMI_MAX_UNITS_NAME);
```

Comment:
Is there a guarantee that that the largest location string is smaller than the largest unit name string?  If not,  using` BMI_MAX_UNITS_NAME` for the buffer size here may be problematic.  Do we need another macro for `BMI_MAX_LOCATION_SIZE` to be explicit?

Reply:
Including a BMI_MAX_LOCATION_NAME may be best practice.  The bmi-examples simply use BMI_MAX_UNITS_NAME as size buffer for a handful of bmi functions defs (e.g. get_var_location).